packages/extension/src/ui/services/useOnClickOutside.ts

```diff
@@ -0,0 +1,30 @@
+import { RefObject, useEffect, useRef } from "react"
+
+export default function useOnClickOutside<T extends HTMLElement>(
```

Comment:
don't use `export default` unless there's no other way
```suggestion
export function useOnClickOutside<T extends HTMLElement>(
```